_episodes/01-introduction.md

```diff
@@ -3,13 +3,241 @@ title: "Introduction"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "What is deep learning?"
+- "When does it make sense to use and not use deep learning?"
+- "When is it successful?"
+- "What are the tools involved?"
+- "What is the workflow for deep learning?"
+- "Why we chose Keras"
 objectives:
-- "First learning objective. (FIXME)"
+- "Recall the sort of problems for which Deep learning is a useful tool"
+- "List some of the available tools for Deep Learning"
+- "Recall the steps of a Deep Learning workflow"
+- "Understand why it is important to test the accuracy of deep learning system."
```

Comment:
Does "explain" imply we feature this in an exercise (we don't currently in this episode).

Reply:
Good question. The switch from "understand" to "explain" is primarily to make this learning objective (a little) more specific and more measurable. 
While I recommend that you have a formative assessment to address every desired outcome from your lesson/episodes, it is not a hard rule. I think you can view it as an acknowledgment that it would be nice to have a way to assess this, but you are not obligated to create that assessment right now. Another way of thinking about this is that it might be an outcome from this introductory episode that is only assessed later in the lesson e.g. when you come to the episode that discusses model accuracy in more detail.